README.rst

```diff
@@ -136,228 +102,140 @@ You can use the ``server_info`` method to get the server information:
     assert 'schema' in info['capabilities'], "Server doesn't support schema validation."
 
 
-Handling buckets
-----------------
-
-All operations are rooted in a bucket. It makes little sense for
-one application to handle multiple buckets at once (but it is possible).
-If no specific bucket name is provided, the "default" bucket is used.
-
-.. code-block:: python
-
-    from kinto_http import Client
-    from kinto_http.patch_type import BasicPatch, MergePatch, JSONPatch
-    credentials = ('alexis', 'p4ssw0rd')
-
-    client = Client(server_url='http://localhost:8888/v1',
-                    auth=credentials)
-
-    # To create a bucket.
-    client.create_bucket(id='payments')
-
-    # To get an existing bucket
-    bucket = client.get_bucket(id='payments')
-
-    # Or retrieve all readable buckets.
-    buckets = client.get_buckets()
-
-    # To create or replace an existing bucket.
-    client.update_bucket(id='payments', data={'description': 'My payments data.'})
-
-    # Or modify some fields in an existing bucket.
-    # The Kinto server supports different types of patches, which can be used from kinto_http.patch_type.
-    client.patch_bucket(id='payments', changes=BasicPatch({'status': 'updated'}))
-
-    # It is also possible to manipulate bucket permissions (see later)
-    client.patch_bucket(id='payments', changes=BasicPatch(permissions={}))
-
-    # Or delete a bucket and everything under.
-    client.delete_bucket(id='payment')
-
-    # Or even every writable buckets.
-    client.delete_buckets()
-
-
-Groups
-------
-
-A group associates a name to a list of principals. It is useful in order to handle permissions.
-
-.. code-block:: python
-
-    # To create a group.
-    client.create_group(id='receipts', bucket='payments', data={'members': ['blah', 'foo']})
-
-    # Or get an existing one.
-    group = client.get_group(id='receipts', bucket='payments')
-
-    # Or retrieve all groups in the bucket.
-    groups = client.get_groups(bucket='payments')
+Bucket operations
+-----------------
 
-    # To create or replace an existing bucket.
-    client.update_group(id='receipts', bucket='payments', 'data'={'members':['foo']})
+* ``get_bucket(id=None, **kwargs)``: retrieve single bucket
+* ``get_buckets(**kwargs)``: retrieve all readable buckets
+* ``create_bucket(id=None, data=None, **kwargs)``: create a bucket
+* ``update_bucket(id=None, data=None, **kwargs)``: create or replace an existing bucket
+* ``patch_bucket(id=None, changes=None, **kwargs)``: modify some fields in an existing bucket
+* ``delete_bucket(id=None, **kwargs)``: delete a bucket and everything under it
+* ``delete_buckets(**kwargs)``: delete every writable buckets
```

Comment:
Maybe we should replace these sections (buckets, groups, collections, records) with Sphinx-generated documentation? I don't think it adds much as-is and it just means we have to replicate the list of methods here every time we change them.

Reply:
Well, I don't know how to do that in a README, if you want to replace the README by Sphinx on readthedocs, that'd be awesome :)